lib/ember-orbit/relationships/has-many.js

```diff
@@ -0,0 +1,22 @@
+const { ArrayProxy } = Ember;
+
+export default ArrayProxy.extend({
+  _store: null,
+  _model: null,
+
+  pushObject(record) {
```

Comment:
Another important reason for doing this: it means that changes received from the underlying LiveQuery won't be reflected back to the Store as new Transforms.